modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -238,10 +238,11 @@ final void readBinary(int bssVersion, DataInputStream is, String[] strings)
     private String[] stringStore;
     final String[] getStringStore() { return stringStore; }
 
-    /** Load a binary stylesheet file from a input stream.
-     * @param url the URL from which the stylesheet will be loaded
-     * @return the constructed stylesheet
-     * @throws java.io.IOException the exception
+    /** Loads a binary stylesheet from a URL.
+     * @param url the URL from which the stylesheet will be loaded.
```

Comment:
No need to add a `.`  because this is a phrase (@param and @return are usually in this form).